include/spirv-tools/optimizer.hpp

```diff
@@ -376,18 +376,6 @@ Optimizer::PassToken CreateCommonUniformElimPass();
 // eliminated with standard dead code elimination.
 Optimizer::PassToken CreateAggressiveDCEPass();
 
-// Creates a pass to consolidate uniform references.
```

Comment:
oh.  Ok. Maybe it was a bad merge on my part.  certainly possible in all my rebasing.